### PR TITLE
en: Add Hooks use-case when changing router.base (related #984)

### PR DESCRIPTION
**Pull request by renoirb**
related #984

### CODE REVIEW

File: en/api/configuration-hooks.md [hunk 1]
```diff
@@ -19,11 +19,14 @@ export default {
...
+          builder.nuxt.options.buildDir,
+          'extra-file'
+        )
         fs.writeFileSync(extraFilePath, 'Something extra')
-      }
-    }
-  }
+      },
+    },
+  },
```
[the remark this answers is not in the document]
Comment (renoirb) [reply en/api/configuration-hooks.md]: No.

It's a manipulation mistake. I've ran prettier on my own changes before committing. This slipped through.

[the remark this answers is not in the document]
Comment (renoirb) [reply en/api/configuration-hooks.md]: I've rebased and reverted this, apologies :)

File: en/api/configuration-hooks.md [hunk 2]
```diff
@@ -36,3 +36,116 @@ Internally, hooks follow a naming pattern using colons (e.g., `build:done`). For
...
+In order to prevent this, you can setup a Hook.
+
+Maybe redirecting is not the best use-case for a production Web site, but this will help you leverage Hooks.
+
+To begin, you [can change `router.base`](/api/configuration-router#base); Update your `nuxt.config.js`:
+
+```js
+// nuxt.config.js
+import hooks from './hooks'
+export default {
```
Comment (clarkdo) [inline en/api/configuration-hooks.md]: Better to keep consistent code style with other pages which has no comma after last property.

**Comment by renoirb** [comment]
I've removed trailing slashes, and rebased.
Should be good now.

File: en/api/configuration-hooks.md (hunk 2 above)
Comment (renoirb) [reply en/api/configuration-hooks.md]: Done!

**Comment by manniL** [comment]
Thank you ☺️

**Pull request merged**